setup_certificates.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
```

Comment:
![image](https://github.com/OpsStrike/dos14-Labzeev_Timofey-git-flow/assets/34745780/6a031ece-2dbb-4731-a2dd-269431c68f8c)

Can we change exit status to a none zero value if we have an error. Because now it exits with 0 status always